ngsild-payloads/applicationPod.jsonld

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "urn:ngsi-ld:ApplicationPod:istio-ingressgateway-9c8b9b586-thl7x",
+  "type": "ApplicationPod",
+  "name": {
+    "type": "Property",
+    "value": "istio-ingressgateway-9c8b9b586-thl7x"
+  },
+  "temporalResolution": {
+    "type": "Property",
+    "value": "PT1H"
+  },
+  "cpuUtilization": {
+    "type": "Property",
+    "observedAt": "2024-05-24T08:00:00Z",
+    "value": 0.069,
+    "unitCode": "P1"
+  },
+  "networkTransmission": {
+    "type": "Property",
+    "observedAt": "2024-05-24T08:00:00Z",
+    "value": 196.1,
+    "unitCode": "byte"
+  },
+  "networkReception": {
+    "type": "Property",
+    "observedAt": "2024-05-24T08:00:00Z",
+    "value": 177.5,
+    "unitCode": "byte"
+  },
+  "ramUtilization": {
+    "type": "Property",
+    "observedAt": "2024-05-24T08:00:00Z",
+    "value": 92.7,
+    "unitCode": "byte"
+  },
+  "diskIo": {
+    "type": "Property",
+    "observedAt": "2024-05-24T08:00:00Z",
+    "value": 14882.1,
+    "unitCode": "byte"
+  },
+  "parent": {
```

Comment:
`deployedInServer` (or just `deployedIn`) seems more meaningful (and you can remove `deployedInNode` from the context if you don't use it anymore)